OpenEdXMobile/res/layout/discussion_response_action_bar_layout.xml

```diff
@@ -2,6 +2,7 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/discussion_actions_bar"
```

Comment:
Also, it would probably be best to have the separate included in this layout, since there's no purpose to it one it's hidden.